canvasapi/course.py

```diff
@@ -733,6 +734,34 @@ def get_assignments(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def get_assignments_for_group(self, assignment_group, **kwargs):
+        """
+        Returns a paginated list of assignments for the given assignment group
+
+        :calls: `GET /api/v1/courses/:course_id/assignment_groups/:assignment_group_id/assignments\
+        <https://canvas.instructure.com/doc/api/assignments.html#method.assignments_api.index>`
+
+        :param assignment_group: The object or id of the assignment group
+        :type assignment_group: :class: `canvasapi.assignment.AssignmentGroup` or int
+
+        :rtype: :class: `canvasapi.paginated_list.PaginatedList` of
+            :class: `canvasapi.assignment.Assignment`
```

Comment:
remove the space between `:class:` and the backtick to properly link a class. Build docs to see.